python/py-pybonjour/Portfile

```diff
@@ -12,12 +12,9 @@ description         pybonjour provides a pure-Python interface to Apple \
                     Bonjour and compatible DNS-SD libraries (such as Avahi).
 long_description    ${description}
 
-homepage            https://code.google.com/p/pybonjour/
-master_sites        googlecode:pybonjour
-distname            pybonjour-${version}
+fetch.type          git
+git.url             https://github.com/DynamicDevices/pybonjour-python3.git
```

Comment:
this should use the [`github` PG](https://guide.macports.org/chunked/reference.portgroup.html#reference.portgroup.github) and use the "proper" upstream (i.e., https://dev.gajim.org/lovetox/pybonjour-python3) instead of a semi-random fork